### PR TITLE
Operator getting shut down because of errors with proof response

### PR DESCRIPTION
**Pull request by foets**
## Motivation

Aleo is amazing idea and project and i wish all bugs to be fixed for it to work properly towards mainnet launch

## Test Plan

(Write your test plan here)

## Related PRs

(Link your related PRs here)
Снимок экрана 2022-01-09 в 6 45 34 PM

I'm not so much technical kind of guy, but as far as i can see, some provers are sending empty proofs back to operator or just don't send any at all and after that operator node can just get crushed. It happened like 5-6 times to me
Снимок экрана 2022-01-09 в 4 54 09 PM

BAsically it's just shutting down the operator node and all related to snarkos or snarkvm activities in the server

### CODE REVIEW

**Comment by ljedrz** [comment]
I think you might have wanted to post this as an issue as opposed to a PR; it's already tracked under https://github.com/AleoHQ/snarkOS/issues/1534. Please feel free to reopen (and fix) if I'm mistaken.
